framework/packages/abstract-std/src/native/ibc_host.rs

```diff
@@ -131,14 +131,23 @@ pub enum ExecuteMsg {
         action: HostAction,
     },
     // ANCHOR_END: ibc-host-execute
-    /// Allows for remote execution from the Polytone implementation on a local module
-    ModuleExecute {
+    /// Allows for remote execution or request from the Polytone implementation on a local module
+    ModuleAction {
         source_module: InstalledModuleIdentification,
         target_module: ModuleInfo,
-        msg: Binary,
+        msg: ModuleActionMsg,
     },
 }
```

Comment:
@CyberHoward ModuleAction inside ModuleAction variant felt weird, went with `ModuleActionMsg`, naming is hard :eyes: